### PR TITLE
rename edm to SearchFieldDataType

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Comment by xiangyan99** [comment]
#11511

File: sdk/search/azure-search-documents/CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
 **Breaking Changes**
 
 - Reorganized `SearchServiceClient` into `SearchIndexClient` & `SearchIndexerClient`    #11507
-- Split searchindex.json and searchservice.json models and operations into separate namespaces #11508
+- Splitted searchindex.json and searchservice.json models and operations into separate namespaces #11508
+- Renamed `edm` to `SearchFieldDataType`    #11511
```
Comment (lmazuel) [inline sdk/search/azure-search-documents/CHANGELOG.md:9]: /azure/search/documents/indexes/_internal/SearchFieldDataType.py

is not a valid Python file name (should be snake case)

**Pull request merged**